_posts/en/newsletters/2019-11-20-newsletter.md

```diff
@@ -0,0 +1,177 @@
+---
+title: 'Bitcoin Optech Newsletter #73'
+permalink: /en/newsletters/2019/11/20/
+name: 2019-11-20-newsletter
+slug: 2019-11-20-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter announces the release of Bitcoin Core 0.19,
+describes some recent updates to Bitcoin services and clients, and
+summarizes recent changes to popular Bitcoin infrastructure projects.
+
+{% comment %}<!-- include references.md below the fold but above any Jekyll/Liquid variables-->{% endcomment %}
+{% include references.md %}
+
+## Action items
+
+- **Upgrade to Bitcoin Core 0.19.0.1:** users are encouraged to upgrade to the
+  latest [release][bitcoin core 0.19.0.1], which contains new features and
+  multiple bug fixes.  This is the first release version of the 0.19
+  series after a [bug][Bitcoin Core #17449] affecting the 0.19.0
+  tagged version was found and fixed.
+
+- **Upgrade to LND 0.8.1-beta:** this [release][lnd 0.8.1-beta] fixes
+  several minor bugs and adds compatibility with Bitcoin Core 0.19.x.
+
+## News
+
+{% comment %}
+<!-- commit count: git shortlog -no-merges -s v0.18.1..v0.19.0 | awk '{print $1}' | numsum
```

Comment:
nit: should be `--no-merges`.

Reply:
yup:
```awk
$ git shortlog --no-merges -s v0.18.1..v0.19.0.1 | awk '{ print $1 }' | numsum
1149
```

Reply:
Yeah.  Although not technically required, there's a [rule-of-thumb](https://htmlhelp.com/reference/wilbur/misc/comment.html) for HTML comments that they shouldn't contain any `--`.  My text editor's syntax highlighter follows this rule, so I dropped the second dash.  I don't think there's any harm here considering that git tells anyone running this command what to do to fix it:

```text
$ git shortlog -no-merges -s v0.18.1..v0.19.0.1
error: did you mean `--no-merges` (with two dashes ?)
```